packages/map/examples/src/map/pages/demo/demo.page.ts

```diff
@@ -60,20 +59,32 @@ public ngOnInit(): void {
 	</div>
 </aui-leaflet>`;
 
-  public mapMockService: MapService = new MapService('browser');
+	public leafletMap: LeafletMap;
 
-	public leafletMap: LeafletMap = new LeafletMap({
-		zoom: 13, // default zoom level
-		center: [51.215, 4.425], // default center point
-		onAddPolygon: (layer) => {},
-		onAddLine: (layer) => {},
-		onEditFeature: (feature) => {},
-	}, this.mapMockService);
+	constructor() {
+		this.leafletMap = new LeafletMap({
```

Comment:
Please don't use the new keyword, but dependency inject an instance using Angular's dependency injection framework.